Fix cart & missing whishlist/comparison products after external login (#3822)
shopsys/shopsys: upgrade-notes/backend_20250224_112831.md

#### Fix cart & missing whishlist/comparison products after external login ([#3822](https://github.com/shopsys/shopsys/pull/3822))

> - for external logins you need to make sure to properly clear the localStorage entries

Reviewer: This is storefront upgrade note right?

Author: I think the changes for `loginAsCustomerUser` template are considered to be BE.

#### Fix cart & missing whishlist/comparison products after external login ([#3822](https://github.com/shopsys/shopsys/pull/3822))

- for external logins you need to make sure to properly clear the localStorage entries

> - when using `loginAsCustomerUser.html.twig` template, you need to manually perform `handleActionsAfterLogin`, mainly reset `localStorage` entries for `cartUuid`, `productListUuids`, `authLoading` and `userEntry`

Reviewer 2: ```suggestion
- if you have extended `loginAsCustomerUser.html.twig` template, you need to manually perform `handleActionsAfterLogin`, mainly reset `localStorage` entries for `cartUuid`, `productListUuids`, `authLoading` and `userEntry`
```

> - see #project-base-diff to update your project

Reviewer 2: in this PR there are no changes in the project-base, so this sentence can be removed
